style/base.css

```diff
@@ -99,3 +99,7 @@ h2 {
   top: 0;
   background-color: white;
 }
+
+.jp-Button:disabled {
```

Comment:
I think this impacts all jyupter lab buttons.

Reply:
Thanks @ptwng  Modified the code to address this.